contracts/UsingWitnet.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.5.0;
+
+import "./WitnetBridgeInterface.sol";
+
+
+
+/**
+ * @title The UsingWitnet contract
+ * @notice Contract writers can inherit this contract in order to create requests for the
+ * Witnet network
+ */
+contract UsingWitnet{
+
+  WitnetBridgeInterface wbi;
+
+  /**
+  * @notice Include an address to specify the WitnetBridgeInterface
+  * @param _wbi WitnetBridgeInterface address
+  */
+  // TODO: Remove this constructor to allow to child contracts define
```

Comment:
This comment may be removed because contracts inheriting this contract may choose their own WBI address [Solidity arguments for base constructors](https://solidity.readthedocs.io/en/develop/contracts.html#arguments-for-base-constructors).